dds/DCPS/transport/rtps_udp/RtpsUdpSendStrategy.cpp

```diff
@@ -293,6 +294,10 @@ RtpsUdpSendStrategy::send_single_i(const iovec iov[], int n,
 {
   OPENDDS_ASSERT(addr != ACE_INET_Addr());
 
+  if (addr == link_->transport().config().rtps_relay_address()) {
+    ++link_->transport().relay_message_counts_.rtps_send;
```

Comment:
Should this only get incremented on a success? Should there be a "failed" count somewhere?